README.md

```diff
@@ -794,38 +794,37 @@ the corresponding package directory can be found inside "packages" directory of
-| `linkPrereleases`  | **optional** Update package versions even if the release is a preview release, "false" by default.                                                                                                                                        |
-| `checksums`        | **optional** A list of checksums that will be computed for matched files (see `includeNames`). Every checksum entry is an object with two attributes: algorithm (one of "sha256", "sha384", and "sha512) and format ("base64" and "hex"). |
-| `onlyIfPresent`    | **optional** A file pattern. The target will be executed _only_ when the matched file is found.                                                                                                                                           |
+| Option | Description                                                                                         |
+| ------ | --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `apps` | List of `app` configs as a dict, keyed by their canonical names (example: `app:craft`)              |
+| `sdks` | List of `sdk` configs as a dict, keyed by their canonical names (example: `maven:io.sentry:sentry`) |
+| `(sdks | apps).urlTemplate`                                                                                  | **optional** URL template that will be used to generate download links for "app" package type.                                                                                                                                             |
+| `(sdks | apps).linkPrereleases`                                                                              | **optional** Update package versions even if the release is a preview release, "false" by default.                                                                                                                                         |
+| `(sdks | apps).checksums`                                                                                    | **optional** A list of checksums that will be computed for matched files (see `includeNames`). Every checksum entry is an object with two attributes: algorithm (one of `sha256`, `sha384`, and `sha512`) and format (`base64` and `hex`). |
+| `(sdks | apps).onlyIfPresent`                                                                                | **optional** A file pattern. The target will be executed _only_ when the matched file is found.                                                                                                                                            |
```

Comment:
Is this GitHub's viewer being wonky, or bad formatting?

(P.S. [Grip](https://github.com/joeyespo/grip) can be useful here.)

<img width="1361" alt="Screen Shot 2021-06-01 at 5 44 17 PM" src="https://user-images.githubusercontent.com/134455/120394178-11720200-c301-11eb-82b1-e6a3f2e58d34.png">

Reply:
It was bad formatting, fixed now! 😁